calico/operations/ebpf/use-cases-ebpf.mdx

```diff
@@ -82,7 +82,7 @@ With eBPF, you can load balance at the source using a program you’ve loaded in
 
 ### ✔ Building a service mesh control plane
 
-Service mesh relies on proxies like Envoy. A lot of thought has gone into designing this process over the years. The main reason for doing it this way is that, in many cases, it is not viable to do inline processing for application protocols like HTTP at the high speeds seen inside a cluster. Therefore, you should think of using eBPF to route traffic to a proxy like Envoy in an efficient way, rather than using it to replace the proxy itself. However, you do need to turn off connect-time load balancing (CTLB) so sidecars can see the service addresses. Given you are already taking a performance hit by the extra hop to the sidecar, not using CTLB performance optimization to avoid NAT overhead is likely not a big deal.
+Service mesh relies on proxies like Envoy. A lot of thought has gone into designing this process over the years. The main reason for doing it this way is that, in many cases, it is not viable to do inline processing for application protocols like HTTP at the high speeds seen inside a cluster. Therefore, you should think of using eBPF to route traffic to a proxy like Envoy in an efficient way, rather than using it to replace the proxy itself. However, you do need to turn off connect-time load balancing (CTLB) so sidecars can see the service addresses. Given you are already taking a performance hit by the extra hop to the sidecar, not using CTLB performance optimization to avoid NAT overhead is likely not a big deal. Set `felixconfiguration` options to `bpfConnectTimeLoadBalancing=Disabled` and `bpfHostNetworkedNATWithoutCTLB=Enabled`.
```

Comment:
I don't think we do have another spot. But in any case, this really isn't the place. Let's remove this from here.

If we need to document how to set up a service mesh control plane, then we should start the process to get that doc written.

Reply:
> However, you do need to turn off connect-time load balancing (CTLB) so sidecars can see the service addresses. Given you are already taking a performance hit by the extra hop to the sidecar, not using CTLB performance optimization to avoid NAT overhead is likely not a big deal.

That piece of information was always there, but incomplete. But well, whatever. Please, write that doc then, otherwise we need to deal with user issues, which takes time and focus away.